src/lib/core/CHIPConfig.h

```diff
@@ -2737,7 +2737,7 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
  * @brief Defines the max number of SessionCreationDelegates
  */
 #ifndef CHIP_CONFIG_MAX_SESSION_CREATION_DELEGATES
-#define CHIP_CONFIG_MAX_SESSION_CREATION_DELEGATES 2
+#define CHIP_CONFIG_MAX_SESSION_CREATION_DELEGATES 4
 #endif
```

Comment:
I think this define may no longer do anything with this commit:

```
commit b210aa3c46ff789918dbca47e2f30227f66f25a1
Author: Zang MingJie <zealot0630@gmail.com>
Date:   Fri Dec 10 22:53:16 2021 +0800
```

CC @kghost

Reply:
Thanks. I removed it and also removes the `#define` that is useless now.